koku/api/migrations/0009_providerstatus_squashed_0042_auto_20200116_2048.py

```diff
@@ -13,16 +13,6 @@
 # Move them and any dependencies into this file, then update the
 # RunPython operations to refer to the local versions:
 
-# api.migrations.0029_cloud_account_seeder
-def seed_cost_management_aws_account_id(apps, schema_editor):
-    """Create a cloud account, using the historical CloudAccount model."""
-    CloudAccount = apps.get_model("api", "CloudAccount")
-    cloud_account = CloudAccount.objects.create(
-        name="AWS", value="589173575009", description="Cost Management's AWS account ID"
-    )
-    cloud_account.save()
-
-
```

Comment:
Don't delete the old migrations. You need to create a new migration.

Reply:
@Red-HAP how would I do that?

Reply:
the migration edit is now reverted.